packages/cli/src/ceramic-daemon.ts

```diff
@@ -393,6 +419,9 @@ export class CeramicDaemon {
     adminModelRouter.getAsync('/', this.getIndexedModels.bind(this))
     adminModelRouter.postAsync('/', this.startIndexingModels.bind(this))
     adminModelRouter.deleteAsync('/', this.stopIndexingModels.bind(this))
+    adminModelDataRouter.getAsync('/', this.getIndexedModelData.bind(this))
+    adminModelDataRouter.postAsync('/', this.startIndexingModelData.bind(this))
+    adminModelDataRouter.deleteAsync('/', this.stopIndexingModels.bind(this))
```

Comment:
this seems weird.  It breaks the pattern and symmetry with the other methods.  I think we should have a real stopIndexingModelData method that takes its input in the same nested object format as the other ModelData methods.  Even if only the `streamId` field of the object is used for now, it still sets up the object structure which would allow us to add new arguments in the future.  Perhaps one day we could use it to delete a specific field index, for example.

If we introduce this endpoint now but with the same API structure as the existing method, then we have two identical endpoints for no gain, plus if we wanted to extend the options available in the future we'd need to add a third endpoint to break the API.